src/Nest/ElasticClient.cs

```diff
@@ -103,6 +147,18 @@ private CatResponse<TCatRecord> DeserializeCatHelpResponse<TCatRecord>(IApiCallD
 			request.RequestParameters.DeserializationOverride = DeserializeCatHelpResponse<CatHelpRecord>;
 			return DoRequestAsync<TRequest, CatResponse<CatHelpRecord>>(request, request.RequestParameters, ct, r => ElasticClient.ForceJson(r));
 		}
+
+		protected NodesHotThreadsResponse DoNodesHotThreads(INodesHotThreadsRequest request)
+		{
+			request.RequestParameters.DeserializationOverride = DeserializeNodesHotThreadsResponse<CatHelpRecord>;
+			return DoRequest<INodesHotThreadsRequest, NodesHotThreadsResponse>(request, request.RequestParameters, r => ElasticClient.ForceJson(r));
+		}
+
+		protected Task<NodesHotThreadsResponse> DoNodesHotThreadsAsync(INodesHotThreadsRequest request, CancellationToken ct)
+		{
+			request.RequestParameters.DeserializationOverride = DeserializeNodesHotThreadsResponse<CatHelpRecord>;
+			return DoRequestAsync<INodesHotThreadsRequest, NodesHotThreadsResponse>(request, request.RequestParameters, ct, r => ElasticClient.ForceJson(r));

```

Comment:
`ForceJson` makes no different with nodes hot threads - I think this should be removed